### PR TITLE
data: Riteshrajcloud

### PR DESCRIPTION
**Pull request by Riteshrajcloud**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Riteshrajcloud.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Ritesh Raj",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a Python Developer",
+  "avatar": "https://github.com/Riteshrajcloud.png"
```
Comment (sital002) [inline data/Riteshrajcloud.json:6]:
```suggestion
```
avatar field isnot required anymore

File: data/Riteshrajcloud.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Ritesh Raj",
+  "displayStatsPublic": true,
+  "type": "personal",
```
Comment (sital002) [inline data/Riteshrajcloud.json:4]:
```suggestion
```
type and displayStatsPublic isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉

**Pull request merged**